.gitignore

```diff
@@ -28,3 +28,4 @@ s101plugin.state
 
 !.idea/checkstyle-idea.xml
 !.idea/externalDependencies.xml
+*.xsd
```

Comment:
Please leave this out of this PR. I'm thinking that we don't want to ignore `.xsd` files since we've got some in the repo, but if you feel like this change is necessary, please open a separate ticket.

Reply:
I understand your comments. 
I will follow your comments.
Thank you.

Reply:
I completed fix my pull request as your comment.
Thanks.